tests/real/community.rs

```diff
@@ -340,3 +341,75 @@ async fn test_jobs_created_with_builder() {
         .unwrap();
     assert!(had_job);
 }
+
+use std::future::Future;
+use std::pin::Pin;
+use tokio::sync::mpsc;
+fn process_hard_task(
+    sender: sync::Arc<mpsc::Sender<bool>>,
+) -> Box<
+    dyn Fn(Job) -> Pin<Box<dyn Future<Output = Result<(), io::Error>> + Send>>
+        + Send
+        + Sync
+        + 'static,
+> {
+    return Box::new(move |j: Job| {
+        let sender = sync::Arc::clone(&sender);
+        Box::pin(async move {
+            let complexity = j.args().iter().next().unwrap().as_u64().unwrap();
```

Comment:
Why not just `j.args()[0]`?

Reply:
I am already doing this automatically, but I agree that since we are not handling `None` properly, direct indexing makes more sence